app/views/notifiers/course/level_notifier/reached/user_notifications/popup.json.jbuilder

```diff
@@ -4,3 +4,11 @@ json.notificationType 'levelReached'
 
 level = notification.activity.object
 json.levelNumber level.level_number
+
+leaderboard_component = current_component_host[:course_leaderboard_component]
+leaderboard_enabled = leaderboard_component.present?
+display_user_count = leaderboard_component.settings.display_user_count
```

Comment:
Sorry just to check if `leaderboard_component` is `nil` then this line will not work right? maybe can do something like:
```ruby
leaderboard_component = ...
if leaderboard_component
  display_user_count = ...
  ranking = ...
end
```

Reply:
Thanks. Fixed and added test!